parsers/yamlparser.go

```diff
@@ -178,14 +178,16 @@ type Application struct {
 }
 
 type DeploymentYAML struct {
-	Application Application `yaml:"application"` //used in deployment.yaml
+	Application Application       `yaml:"application"` //used in deployment.yaml
+    Packages   map[string]Package `yaml:"packages"` //used in deployment.yaml
```

Comment:
Unfortunately, we should support "Package" (singular) here as well if we want to cover all combinations (even though we see how deploymentreader.go is getting a lot of if-else logic to allow for all these combinations).

Reply:
done